src/Sulu/Component/Content/Tests/Unit/Repository/ContentRepositoryTest.php

```diff
@@ -110,6 +111,10 @@ public function setUp(): void
         $this->sessionManager = $this->prophesize(SessionManagerInterface::class);
         $this->documentManager = $this->prophesize(DocumentManagerInterface::class);
         $this->propertyEncoder = $this->prophesize(PropertyEncoder::class);
+        $this->propertyEncoder->localizedContentName(Argument::type('string'), Argument::type('string'))
+            ->will(fn ($args) => 'i18n-'.$args[1].'-'.$args[0]);
+        $this->propertyEncoder->systemName('order')->willReturn('order');
```

Comment:
This was missing from the test and just was never caught in the older versions of the Query as PHPCR didn't check for null arguments